lib/tasks/azure.rake

```diff
@@ -1,10 +1,30 @@
-#namespace :manageiq do
-#  namespace :providers do
-#    namespace :azure do
-#      desc "Explaining what the task does"
-#      task :your_task do
-#        # Task goes here
-#      end
-#    end
-#  end
-#end
+namespace :manageiq do
+  namespace :providers do
+    namespace :azure do
+      namespace :spec do
+        namespace :regenerate do
+          base_dir = File.join(ManageIQ::Providers::Azure::Engine.config.root.to_s, 'spec')
+          cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/azure')
+          spec_dir = File.join(base_dir, 'models/manageiq/providers/azure')
+
+          desc "Regenerate all the cassettes"
+          task :all do
+            spec_dir = File.expand_path("../../spec", __dir__)
```

Comment:
I'm just curious, where's the `spec_dir` used in this task?

Reply:
It's not used, let me clean that up.